Makefile

```diff
@@ -305,5 +305,10 @@ env: ## Generate env files
 	@echo "$(P) Generate/Regenerate env files required for application (generated passwords only match if database .env file does not already exist)"
 	@./scripts/gen-env-files.sh;
 
-.PHONY: logs start destroy local setup restart refresh up down stop build rebuild clean client-test server-test pause-30 server-run db-clean db-drop db-seed db-refresh db-script db-scaffold npm-clean npm-refresh keycloak-sync convert backend-coverage frontend-coverage backend-test frontend-test env
+mayan-up: ## Calls the docker compose up for the mayan images
+	@echo "$(P) Create or start mayan-edms system"
+	@cd tools/mayan-edms
+	@docker-compose up -d
```

Comment:
Not a big deal but this two lines should be made into one. Make executes each command/line in a new bash shell so the "cd" command does nothing for docker-compose. See other targets in this file like **frontend-test** for reference

Reply:
good catch, I've updated it